contents/docs/experiments/creating-an-experiment.mdx

```diff
@@ -44,43 +44,27 @@ Each experiment is backed by a [feature flag](/docs/feature-flags/manual). In yo
 
 The feature flag will be created for you automatically. Therefore, you must add a feature flag key which does not exist yet.
 
-> **Advanced:** It's possible to create experiments without using PostHog's feature flags (for example, if you you're a different library for feature flags). For more info, read our docs on [implementing experiments without feature flags](/docs/experiments/running-experiments-without-feature-flags) 
+> **Advanced:** It's possible to create experiments without using PostHog's feature flags (for example, if you're using a different third-party library for feature flags). For more info, read our docs on [implementing experiments without feature flags](/docs/experiments/running-experiments-without-feature-flags) 
 
 ## Experiment variants
 
-By default, all experiments have a 'control' and 'test' variant. If you have more variants that you'd like to test, you can use this field to add more variants and run multivariant tests.
+By default, all experiments have a `control` and `test` variant. You can add more variants if you like.
 
-Participants are automatically split equally between variants. If you want assign specific users to a specific variant, you can set up [manual overrides](/docs/experiments/common-questions#how-do-i-assign-a-specific-person-to-the-controltest-variant-in-an-experiment).
+Participants are automatically split equally between variants. If you want assign specific users to a specific variant, see the section below on [distribution and release conditions](#distribution-and-release-conditions).
 
 ## Participant type
 
-The default is users, but if you've created [groups](/docs/product-analytics/group-analytics), you can run [group-targeted experiments](/blog/running-group-targeted-ab-tests). This will test how a change affects your product at a group-level by providing the same variant to every member of a group.
-
-## Participant targeting
-
-By default, your experiment will target 100% of participants. If you want to target a more specific set of participants, or change the rollout percentage, you need to do this by changing the [release conditions](/docs/feature-flags/creating-feature-flags#release-conditions) for the feature flag used by your experiment.
-
-Below is a video showing how to navigate there:
-
-<ProductVideo
-    videoLight={SetExperimentLight} 
-    videoDark={SetExperimentDark}
-    alt="How to create an experiment in PostHog" 
-    classes="rounded"
-/>
+By default PostHog runs experiments at a user-level. This means that participants are assigned to a variant based on their user properties. However, if you've created [groups](/docs/product-analytics/group-analytics), you can also run [group-targeted experiments](/blog/running-group-targeted-ab-tests). This will test how a change affects your product at a group-level by providing the same variant to every member of a group.
```

Comment:
```suggestion
By default, PostHog runs experiments at a user-level. This means that participants are assigned to a variant based on their user properties. However, if you've created [groups](/docs/product-analytics/group-analytics), you can also run [group-targeted experiments](/blog/running-group-targeted-ab-tests). This will test how a change affects your product at a group-level by providing the same variant to every member of a group.
```